includes/functions.php

```diff
@@ -3675,3 +3673,135 @@ function erp_is_timestamp( $string ) {
 
     return true;
 }
+
+/**
+ * Validates people's name
+ *
+ * @since 1.6.7
+ *
+ * @param string $name
+ *
+ * @return bool
+ */
+function erp_is_valid_name( $name ) {
+    return ctype_alpha( iconv( 'UTF-8', 'ISO-8859-1', preg_replace( '/\s/', '', $name ) ) );
+}
+
+/**
+ * Validates company's name
+ *
+ * @since 1.6.7
+ *
+ * @param string $company
+ *
+ * @return bool
+ */
+function erp_is_valid_company( $company ) {
+    return preg_match( '/^[A-Z0-9][ ._\-A-Z0-9]+$/i', $company );
+}
+
+/**
+ * Validates customer's age
+ *
+ * @since 1.6.7
+ *
+ * @param string $age
+ *
+ * @return bool
+ */
+function erp_is_valid_age( $age ) {
+    if ( preg_match( '/^[1-9]{2,3}$/', $age ) ) {
+        if ( intval( $age ) >= 16 && intval( $age ) <= 150 ) {
```

Comment:
Do we have an age restriction anywhere ?

Reply:
No but I thought it could be added. Should I remove the restriction?